core/field_image.ts

```diff
@@ -12,21 +12,22 @@
 import * as goog from '../closure/goog/goog.js';
 goog.declareModuleId('Blockly.FieldImage');
 
-import {FieldConfig, Field} from './field.js';
+import {Field, FieldConfig, FieldValidator} from './field.js';
 import * as fieldRegistry from './field_registry.js';
 import * as dom from './utils/dom.js';
 import * as parsing from './utils/parsing.js';
 import type {Sentinel} from './utils/sentinel.js';
 import {Size} from './utils/size.js';
 import {Svg} from './utils/svg.js';
 
+export type FieldImageValidator = FieldValidator<string>;
```

Comment:
Unused.

Reply:
Ope! Good catch! Looks like `FieldImage` isn't utilizing a validator type at all.

Another similar yet different case - `core/field_textinput` technically isn't using `FieldTextInputValidator` on its own, though that type works for the provided validator for the constructor. I went back and forth on if `Field` subclasses with validators should all have their own validator functions anyway, even if not called out in their constructors. In the case of `FieldTextInput`, the intellisense will show up as asking for `FieldInputValidator<string>`. It seems like an intuitive pattern to keep with, though - expecting the corresponding type in the corresponding class.

Reply:
> Another similar yet different case - core/field_textinput technically isn't using FieldTextInputValidator on its own, though 
that type works for the provided validator for the constructor.

Hmm, for that case, do you think you could just define a pass-through constructor that uses the `FieldTextInputValidator` type? I don't think there's any harm in doing so is there?

Reply:
I think the trade off is that there's another point of maintenance for comments, though that's pretty minor. Some of the comments should probably be tailored to the individual class anyway - I'll go ahead and make the update!

Reply:
done!